--- pep8-1.7.1/testsuite/test_all.py.orig	2017-10-24 16:38:06.000000000 +0200
+++ pep8-1.7.1/testsuite/test_all.py	2025-04-16 20:01:31.834474675 +0200
@@ -49,11 +49,12 @@ def suite():
     from testsuite import test_api, test_parser, test_shell, test_util
 
     suite = unittest.TestSuite()
-    suite.addTest(unittest.makeSuite(Pep8TestCase))
-    suite.addTest(unittest.makeSuite(test_api.APITestCase))
-    suite.addTest(unittest.makeSuite(test_parser.ParserTestCase))
-    suite.addTest(unittest.makeSuite(test_shell.ShellTestCase))
-    suite.addTest(unittest.makeSuite(test_util.UtilTestCase))
+    testLoader = unittest.TestLoader()
+    suite.addTest(testLoader.loadTestsFromTestCase(Pep8TestCase))
+    suite.addTest(testLoader.loadTestsFromTestCase(test_api.APITestCase))
+    suite.addTest(testLoader.loadTestsFromTestCase(test_parser.ParserTestCase))
+    suite.addTest(testLoader.loadTestsFromTestCase(test_shell.ShellTestCase))
+    suite.addTest(testLoader.loadTestsFromTestCase(test_util.UtilTestCase))
     return suite
 
 
